.github/workflows/package_monitor.yml

```diff
@@ -0,0 +1,25 @@
+name: Auto instrumentation for version updates
```

Comment:
Could the expression "auto instrumentation" cause any kind of serious confusion for readers?
I wonder if people would understand this as something else, like a no-code feature or something like this.
Otherwise, please disregard my comment.

Reply:
How about "Currency"? ;) 

Do you have any suggestion?

Reply:
hahaha funny man!
good question. I did not think of any suggestions. Something around "Auto release for version updates?" or "Auto versioning update for instrumentations". 
It's not a big deal also. We can think of a new name if someone asks about auto instrumentation hehe

Reply:
Changed it to "Auto update for instrumentation packages' for the time being.